api/v1beta1/ocicluster_conversion.go

```diff
@@ -86,9 +86,10 @@ func convertv1beta2NSGListTov1beta1NSGList(in []*v1beta2.NSG) ([]*NSG, error) {
 
 func Convertv1beta1AdMapTov1beta2AdMap(in map[string]OCIAvailabilityDomain) (map[string]v1beta2.OCIAvailabilityDomain, error) {
 	out := make(map[string]v1beta2.OCIAvailabilityDomain)
-	for k, v := range in {
+	for k, _ := range in {
 		outV := &v1beta2.OCIAvailabilityDomain{}
-		err := Convert_v1beta1_OCIAvailabilityDomain_To_v1beta2_OCIAvailabilityDomain(&v, outV, nil)
+		inDomain := in[k]
```

Comment:
more of a question than a request for change: why not use `&in[k]` in the below call like what is done in cloud/scope/machine_pool.go